sdks/java/core/src/main/java/org/apache/beam/sdk/schemas/utils/JsonUtils.java

```diff
@@ -26,8 +26,58 @@
 import org.apache.beam.sdk.util.RowJson;
 import org.apache.beam.sdk.util.RowJsonUtils;
 import org.apache.beam.sdk.values.Row;
+import org.everit.json.schema.ArraySchema;
+import org.everit.json.schema.NumberSchema;
+import org.everit.json.schema.ObjectSchema;
+import org.everit.json.schema.ReferenceSchema;
+import org.json.JSONObject;
 
-/** Utils to convert JSON records to Beam {@link Row}. */
+/**
+ * Utils to convert JSON records to Beam {@link Row}.
+ *
+ * <h2>JSON-Schema (https://json-schema.org) support</h2>
+ *
+ * <p>This class provides utility methods to parse, validate and translate between <b>JSON
+ * Schema</b>-formatted schemas and Beam Schemas. The support is based on the <code>
+ * everit-json-schema</code> package, which is <b>not provided by default</b>.
+ *
+ * <p>Therefore, functionality in {@link JsonUtils::beamSchemaFromJsonSchema} requires that you
+ * include {@code everit-json-schema} in your project like so:
+ *
+ * <pre>{@code
+ *  <dependency>
+ * 	  <groupId>com.github.erosb < /groupId>
+ * 	  <artifactId>everit-json-schema < /artifactId>
+ * 	  <version>1.14.1 < /version>
```

Comment:
```suggestion
 *  <dependency>
 * 	  <groupId>com.github.erosb</groupId>
 * 	  <artifactId>everit-json-schema</artifactId>
 * 	  <version>1.14.1</version>
```

Reply:
I had to leave these spaces because the javadoc plugin was catching the `</...>` tags as unmatched html tags (even though they're inside a `{@code ..}` block). I'll leave this as is if that's ok.